### PR TITLE
fix : prevent exiting episode range by using previous episode

### PR DESCRIPTION
**Pull request by Goldenbeasty**
Prevent user escaping episode range by
1. Run `ani-cli evangelion`
2. Choose `1`
3. Choose `1`
4. Choose `p` (previous episode)

### CODE REVIEW

File: ani-cli [hunk 1]
```diff
@@ -382,15 +382,16 @@ open_episode () {
...
 	is_number "$ep_choice_start"
 		if [ "$last_ep_number" -eq 0 ]; then
 			die "Episodes not released yet!"
 		else
 			err "Episode out of range"
 		fi
 		prompt "Choose episode" "[$first_ep_number-$last_ep_number]"
-		episode="$REPLY $REPLY2"
+		episode="$REPLY"
+		ep_choice_start=$episode
```
Comment (ura43) [inline ani-cli:394]: This probably breaks range selection

**Review by ura43 — CHANGES_REQUESTED** [review]

File: ani-cli (hunk 1 above)
Comment (Goldenbeasty) [reply ani-cli]: If I understand correctly range selection is not possible with episode variable anyways correct me if I'm wrong

**Comment by RaynardGerraldo** [comment]
im sorry but we dont need this, ani-cli doesnt even show `[p] previous episode` when playing the first episode

plus ,users are not going to use the previous episode option when they know they are playing the first episode

File: ani-cli (hunk 1 above)
Comment (BAaboe) [reply ani-cli]: this breaks the range selection

**Comment by ura43** [comment]
Sorry but this seems pointless

**Pull request closed without merging**